php/libraries/NDB_Client.class.inc

```diff
@@ -133,17 +133,45 @@ class NDB_Client
             . $config_additions
         );
         // start php session
-        session_start(
-            ['cookie_httponly' => true]
-        );
+        if (PHP_MAJOR_VERSION === 7 && PHP_MINOR_VERSION >= 3) {
+            session_start(
+                array(
+                 'cookie_httponly' => true,
+                 'cookie_samesite' => true,
+                )
+            );
+        } else {
+            // TODO Remove this branch when 7.3 is the minimum required
+            // version for LORIS.
+            session_start(
+                array('cookie_httponly' => true)
+            );
+        }
 
         // if exiting, destroy the php session
         if (isset($_REQUEST['logout']) && $_REQUEST['logout']) {
             session_destroy();
             header("HTTP/1.1 303 See Other");
             header("Location: /");
+            if (PHP_MAJOR_VERSION === 7 && PHP_MINOR_VERSION >= 3) {
+                session_start(
+                    array(
+                     'cookie_httponly' => true,
+                     'cookie_samesite' => true,
+                    )
+                );
+            } else {
+                // TODO Remove this branch when 7.3 is the minimum required
+                // version for LORIS.
+                session_start(
+                    array('cookie_httponly' => true)
+                );
+            }
             session_start(
-                ['cookie_httponly' => true]
+                array(
+                 'cookie_httponly' => true,
+                 'cookie_samesite' => true,
+                )
```

Comment:
Your starting 2 sessions. The lines 170-174 are executed after the if (PHP_VERSION) at line 156 (both conditions)
remove lines 170-174